tools/infrastructure/git-hooks/pre-commit

```diff
@@ -1,5 +1,29 @@
 #!/bin/bash
 
+TEXT_DEFAULT="\\033[0;39m"
+TEXT_INFO="\\033[1;32m"
+TEXT_ERROR="\\033[1;31m"
+TEXT_UNDERLINE="\\0033[4m"
+TEXT_BOLD="\\0033[1m"
+
+PYTHON_FILES_NUMBER=$(git diff --name-only --cached | grep -e '\.py$' | wc -l)
```

Comment:
@dev-gh, please, move validation to the /infrastucture/checkstyle.sh. Now all actual checks are there. New logic is next. pre-commit hook just calls this style checker. So if we will add some more checks in the future, then there will be no need to re-install hook (this is usually not done by devs). We can just update style checker and automatically hook will use new version.